### PR TITLE
open PR to get tag-aligner alab article into staging

### PR DESCRIPTION
**Pull request by rennis250**

### CODE REVIEW

File: alpha-lab/tag-aligner/index.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+## Real-world Positions and Rotations
+
+![Comparison of the different ways to project Neon data into various coordinate systems](./coord-sys-comparisons.png)
+
+It's often important to know the position of an observer in an environment, and how they orient their head and gaze while navigating it. For example, to understand when museum-goers pause in thought while viewing artworks, or how engineers move about and gaze at the surrounding environment during mission-critical tasks.
+
+With a digital twin of a real-world environment, you can visualize an observer's trajectory, and how they direct their gaze spatially onto objects within the scene, by mapping Neon’s gaze + head pose into the digital twin.
+
+In this guide, we'll show you how to do this using data from Neon + Reference Image Mapper (RIM) and the digital twin of your choice.
+
```
Comment (N-M-T) [inline alpha-lab/tag-aligner/index.md:44]: @rennis250, add a note here, something like:

If you're not already familiar with our RIM enrichment, be sure to check out the [RIM documentation]()

File: alpha-lab/tag-aligner/index.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+## Transforming Poses from RIM Data
+
+The first step, as in the title, is to transform camera poses from the RIM 3D model to the Digital twin. For context, our RIM enrichment uses 3D features of a scene to map gaze onto a reference image regardless of the subject’s position or orientation. Under the hood, RIM builds a sparse 3D model of the environment and calculates camera poses relative to the 3D features of the scene. However, the origin, scaling, and orientation of the coordinate system for these camera poses is arbitrary and cannot be used directly for any real-world metrics or visualizations.
+
+![Depiction of the sparse 3D model produced by our Reference Image Mapper](./rim_3d_model.png)
+
+The white dots on this image (statue of [Theodor Koch-Grünberg](https://en.wikipedia.org/wiki/Theodor_Koch-Grunberg)) represent key points of a sparse 3D model built from a RIM enrichment scanning recording. The model is used by RIM to calculate scene camera positions in an arbitrary coordinate system.
```
Comment (N-M-T) [inline alpha-lab/tag-aligner/index.md:51]: Shouldn't this be a caption of the image? Can you make it a caption?

File: alpha-lab/tag-aligner/index.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+After calculating the transformation to convert from the RIM enrichment’s arbitrary coordinate system to your desired coordinate system, the transformation can then be applied to all of the recordings’ poses from that enrichment.
+
+These aligned poses could be used for analysis or to visualize observer motion and gaze within a digital twin. The Tag Aligner package includes a simple 3D viewer that can load a glTF model and allow you to explore the scene while playing back the recording. A set of Neon frames and the gaze ray are also added to the scene for visualization.
+
+## Steps to recreate
+
+1. AprilTags are the key to Tag Aligner, hence the name, so make sure you have one printed and at the ready! You will want a tag from the “tag36h11” family, printed at a good, visible size. We have already prepared [a PDF of them](https://github.com/pupil-labs/pupil-helpers/blob/master/markers_stickersheet/tag36h11_full.pdf?raw=True) for you. Note that you need to include a white border around the printed AprilTag.
```
Comment (N-M-T) [inline alpha-lab/tag-aligner/index.md:61]: Good, visible size is somewhat arbitrary. Recommend changing to something like,

"The marker will need to be clearly visible to the scene camera for this tool to work, so you'll need to think about the necessary size of the marker for your environment"

File: alpha-lab/tag-aligner/index.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+3. If you have a glTF model of your environment and want to visualize the aligned poses and gaze, then be sure to check out the “Bonus” section of the Tag Aligner repo, where we offer a real-time visualization, a Blender plugin, and a Python notebook with some basic analysis.
+
+
+## Working with Aligned Poses
+
+You have now expanded the analysis possibilities of Neon + RIM to the third dimension!
+
+After running the Tag Aligner tool, you will find a file in the recording folder, called "aligned_poses.csv", with the scaled and aligned poses of the scene camera over time.
+
+If you run the bonus section, you will also have a pop-up interactive window that renders a glTF model of your environment that you can use to visualize the aligned poses and gaze.
```
Comment (N-M-T) [inline alpha-lab/tag-aligner/index.md:72]: Shouldn't you mention blender here as well?

**Review by N-M-T — CHANGES_REQUESTED** [review]

File: alpha-lab/tag-aligner/index.md (hunk 1 above)
Comment (rennis250) [reply alpha-lab/tag-aligner/index.md:44]: done

File: alpha-lab/tag-aligner/index.md (hunk 3 above)
Comment (rennis250) [reply alpha-lab/tag-aligner/index.md:61]: done

File: alpha-lab/tag-aligner/index.md (hunk 4 above)
Comment (rennis250) [reply alpha-lab/tag-aligner/index.md:72]: sure. done.

**Pull request merged**